lib/ezutils/classes/ezexecution.php

```diff
@@ -177,7 +177,7 @@ static public function registerShutdownHandler( $documentRoot = false )
      * @params Exception the exception
```

Comment:
Shouldn't we modify this comment like in the original PR?
"* @params Exception|Throwable the exception"